### PR TITLE
stolostron/backlog#22884 cross arch rel img

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Review by berenss — APPROVED** [review]

**Review by Randy424 — APPROVED** [review]
/lgtm

File: clusters/release_images.adoc [hunk 1]
```diff
@@ -56,3 +56,100 @@ The `subscription` directory contains files that specify where the list of relea
...
+
+You can create a cluster on an architecture that is different from the architecture of the hub cluster by manually creating a release image that contains the files for both architectures. 
+
+For example, you might need to create an `x86_64` cluster from a hub cluster that is running on the `ppc64le`, `aarch64`, or `s390x` architecture. If you create the release image with both sets of files, the cluster creation succeeds because the new release image enables the {ocp-short} release registry to provide a multi-architecture image manifest. 
+
+To create the release image, complete steps similar to the following example for your architecture type:
+
+. From the https://quay.io/repository/openshift-release-dev/ocp-release[{ocp-short} release registry], create a https://docs.docker.com/registry/spec/manifest-v2-2/[manifest list] that includes `x86_64`, `s390x`, `aarch64`, and `ppc64le` release images.
+
+.. Pull the manifest lists for both architectures in your environment from the https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay repository] using the following example commands:
```
Comment (dockerymick) [inline clusters/release_images.adoc:71]: I usually add `link:` in front of the url, but either way is fine.

File: clusters/release_images.adoc [hunk 2]
```diff
@@ -56,3 +56,100 @@ The `subscription` directory contains files that specify where the list of relea
...
+$ podman pull quay.io/openshift-release-dev/ocp-release:4.10.1-x86_64
+$ podman pull quay.io/openshift-release-dev/ocp-release:4.10.1-ppc64le
+$ podman pull quay.io/openshift-release-dev/ocp-release:4.10.1-s390x
+$ podman pull quay.io/openshift-release-dev/ocp-release:4.10.1-aarch64
+----
+
+.. Log in to your private repository where you maintain your images:
++
+----
+$ podman login <private-repo>
```
Comment (dockerymick) [inline clusters/release_images.adoc]: I mentioned to Oliver that the `$` before the command. I mentioned that we should mention this during the doc architect call

File: clusters/release_images.adoc [hunk 3]
```diff
@@ -56,3 +56,100 @@ The `subscription` directory contains files that specify where the list of relea
...
++
+----
+$ podman manifest push mymanifest docker://<private-repo>/ocp-release:4.10.1
+----
++
+Replace `private-repo` with the path to your repository.
+
+. On the hub cluster, create a release image that references the manifest in your repository.
+
+.. Create a `YAML` file that contains information that is similar to the following example:
```
Comment (dockerymick) [inline clusters/release_images.adoc]: remove the monospace from YAML. If you plan to use the monospace when YAML is mentioned as a file extension: `.yaml`

File: clusters/release_images.adoc [hunk 4]
```diff
@@ -56,3 +56,100 @@ The `subscription` directory contains files that specify where the list of relea
...
++
+Replace `private-repo` with the path to your repository.
+
+.. Run the following command on your hub cluster to apply the changes:
++
+----
+oc apply -f <file-name>.yaml
+----
++
+Replace `file-name` with the name of the `YAML` file that you just created.
```
Comment (dockerymick) [inline clusters/release_images.adoc]: Same suggestion

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made a few suggestions

File: clusters/release_images.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/release_images.adoc:71]: Thanks. I thought that we came to the conclusion during a meeting that we will leave "link" on the ones that exist, but moving forward, we would not use link. Both work, but let me know if that is wrong for the sake of consistency.

File: clusters/release_images.adoc (hunk 2 above)
Comment (cadawson) [reply clusters/release_images.adoc]: I normally don't leave it. I can remove it from here. Thanks!

File: clusters/release_images.adoc (hunk 3 above)
Comment (cadawson) [reply clusters/release_images.adoc]: Updated

File: clusters/release_images.adoc (hunk 4 above)
Comment (cadawson) [reply clusters/release_images.adoc]: Updated. Thanks!

File: clusters/release_images.adoc (hunk 1 above)
Comment (dockerymick) [reply clusters/release_images.adoc:71]: That's cool with me!

**Review by dockerymick — APPROVED** [review]
Thanks for accepting my suggestion Chris! Approving

**Pull request merged**